esphome/components/udp/__init__.py

```diff
@@ -38,6 +41,21 @@
 )
 
 
+def _final_validate(config):
+    enable_ipv6 = fv.full_config.get().get("network").get("enable_ipv6")
```

Comment:
Please use consts here. `CONF_NETWORK` does not yet exist
```suggestion
    enable_ipv6 = fv.full_config.get().get(CONF_NETWORK).get(CONF_ENABLE_IPV6)
```